docs/source/conf.py

```diff
@@ -87,7 +87,6 @@
     # roocs
     "clisops",
     "daops",
```

Comment:
`daops` is excluded from the `environment.yml` but not here. Is it needed?

Reply:
This is causing the Docker build failure.

Reply:
... need to wait for the latest `daops` build to include it in the `environment.yml`. The docker build relies on conda dependencies only.

Reply:
done. conda daops package works in docker build test.